src/styles/common/_colors.scss

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) 2021 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+// Dell Design Standard Color codes
+
+$primary-white: #FFFFFF;
+$gray-100: #F7F7F7;
+$gray-200: #EEEEEE;
+$gray-300: #E4E4E4;
+$gray-450:  #BBBBBB;
+$gray-550: #888888;
+$gray-700: #666666;
+$gray-900: #333333;
+
+$blue-50: #E7F2FB;
```

Comment:
```suggestion
$blue-400: #E7F2FB;
```

Reply:
Same as above, there are many shades for each color so sticking with DDS convention https://web.ddsproduct.com/31b3fd8b1/p/60290f-colors